contracts/fungible-tokens/andromeda-lockdrop/src/contract.rs

```diff
@@ -67,7 +67,7 @@ pub fn instantiate(
         deposit_window: msg.deposit_window,
         withdrawal_window: msg.withdrawal_window,
         lockdrop_incentives: Uint128::zero(),
-        incentive_token: msg.incentive_token,
+        incentive_token: msg.incentive_token.get_raw_address(&deps.as_ref())?,
```

Comment:
We should store this as `AndrAddr` and resolve it whenever it is required.

Reply:
Changed to save AndrAddr